Components/rating.html

```diff
@@ -0,0 +1,69 @@
+            <a href = "install.html">Installation</a>
+            <p>Components</p>
+            <ul class = "flex flex-column gap-xl">
+                <li><a href="avatar.html">Avatar</a></li>
+                <li><a href="badge.html">Badge</a></li>
+                <li><a href="alert.html" >Alert</a></li>
+                <li><a href="button.html">Button</a></li>
+                <li><a href="card.html">Cards</a></li>
+                <li><a href="input.html">Input</a></li>
+                <li><a href="image.html">Image</a></li>
+                <li><a href="navigation.html">Navigation</a></li>
+                <li><a href="slider.html">Slider</a></li>
+                <li><a href="drawer.html">Drawer/List</a></li>
+                <li><a href="modal.html">Modal</a></li>
+                <li><a href="toast.html">Toast</a></li>
+                <li><a href="rating.html" class = "active-link">Rating</a></li>
+            </ul>
+        </aside>
+
+        <div class = "flex flex-column main gap-m debug">
+            <h1>Rating Component</h1>
+            <p>Product ratings are the star ratings that customers give to your products.</p>
+            <p class = "m1-top">Product Rating is a fantastic way to show the popularity of your product or service and encourage customers to purchase these items.</p>
+
+        <div class = "container flex light-border" style = "align-items: center;max-width:100%">
+            
+                <div>
+                    <i class="material-icons md-32 checked-rating">star</i>
+                    <i class="material-icons md-32 checked-rating">star</i>
+                    <i class="material-icons md-32 checked-rating">star</i>
+                    <i class="material-icons md-32 checked-rating">star</i>
+                    <i class="material-icons md-32 unchecked-rating">star</i>
+                </div>
+
+        </div>
+        
+
+        <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%253E%250A%2509%253Ci%2520class%253D%2522material-icons%2520md-32%2520checked-rating%2522%253Estar%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522material-icons%2520md-32%2520checked-rating%2522%253Estar%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522material-icons%2520md-32%2520checked-rating%2522%253Estar%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522material-icons%2520md-32%2520checked-rating%2522%253Estar%253C%252Fi%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522material-icons%2520md-32%2520unchecked-rating%2522%253Estar%253C%252Fi%253E%250A%253C%252Fdiv%253E"
+            style="width: 649px; height: 297px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
```

Comment:
if you can make a separate class for iframe styling it will be good just to avoid inline styling